e2e/test/scenarios/dashboard-filters/old-parameters.cy.spec.js

```diff
@@ -94,6 +96,24 @@ describe("scenarios > dashboard > OLD parameters", () => {
       cy.findByText("Rows 1-6 of 200").should("be.visible");
       cy.button("Apply").click();
       cy.findByText("Rows 1-6 of 53").should("be.visible");
+
+      // test that the apply button won't showup even when adding a new parameter or removing existing ones
```

Comment:
This tests the first issue Maz found in https://github.com/metabase/metabase/pull/30232#issuecomment-1515419707